src/extension.ts

```diff
@@ -377,7 +383,7 @@ function registerCommands(
   );
 
   context.subscriptions.push(
-    commands.registerCommand("sourcery.chat_request", (message: string) => {
+    commands.registerCommand("sourcery.chat_request", (message) => {
```

Comment:
this isn't actually a string - added a linear issue to refactor and define the messages we pass around https://linear.app/sourcery/issue/SOU-1435/refactor-improve-typing-and-message-passing-in-extension